java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -322,6 +324,29 @@ public void set_only_if_does_not_exists_missing_key(BaseClient client) {
         assertEquals(ANOTHER_VALUE, data);
     }
 
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void set_get_binary_data(BaseClient client) {
+        GlideString key = gs("set_get_binary_data_key");
+        byte[] binvalue = {(byte) 0x01, (byte) 0x00, (byte) 0x01, (byte) 0x00, (byte) 0x02};
+        assert client.set(key, gs(binvalue)).get().equals("OK");
```

Comment:
Please use `assertEquals` in tests to have clearer error messages and test reports